homeassistant/core.py

```diff
@@ -432,7 +432,7 @@ def async_fire(self, event_type: str, event_data=None,
         """
         if event_type != EVENT_HOMEASSISTANT_STOP and \
                 self._hass.state == CoreState.stopping:
-            raise ShuttingDown('Home Assistant is shutting down.')
+            raise ShuttingDown("Home Assistant is shutting down")
```

Comment:
You're sometimes switching quotes from single to double, and sometimes from double to single. What's going on ?